src/components/Calculator.css

```diff
@@ -10,6 +10,7 @@
     font-family: cursive, sans-serif;
     border: 2px solid black;
     border-bottom: none;
+    background-color: grey;
   }
   
   .screen {
```

Comment:
- [ ] [OPTIONAL] The numbers on your screen are almost collapsing into the border. To fix this, you can add a little bit of padding to the right side of the screen. This will give the numbers some breathing room and make them easier to read.